src/Chirp.Razor/Pages/UserTimeline.cshtml

```diff
@@ -30,4 +32,5 @@
     {
         <em>There are no cheeps so far.</em>
     }
+    <small>page @Model.PageNumber</small>
```

Comment:
add buttons here to change pages

Reply:
Does 64381b11297597d1d2ec9577780e7b80e1754170 solve this?

Reply:
Yep ☺️